finagle-http/src/main/scala/com/twitter/finagle/http/filter/StatsFilter.scala

```diff
@@ -23,6 +23,12 @@ class StatsFilter[REQUEST <: Request](stats: StatsReceiver)
   private[this] val timeReceiver = stats.scope("time")
   private[this] val responseSizeStat = stats.stat("response_size")
 
+  private[this] val counterCache: Map[String, Counter] =
+    Map.empty.withDefault(statusReceiver.counter(_))
```

Comment:
Oh geez, it might not be open source.  Let me check if I just got the name wrong or if we haven't open sourced it yet.